src/transformers/__init__.py

```diff
@@ -1212,6 +1213,18 @@
             "ErniePreTrainedModel",
         ]
     )
+    _import_structure["models.esm"].extend(
+        [
+            "ESM_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "ESMForCausalLM",
+            "ESMForMaskedLM",
+            "ESMForSequenceClassification",
+            "ESMForTokenClassification",
+            "ESMLayer",
```

Comment:
```suggestion
```
Let's leave the layer inside models.esm only. It doesn't need to be in the main init.